lte/gateway/c/session_manager/UpfMsgManageHandler.cpp

```diff
@@ -70,4 +71,56 @@ void UpfMsgManageHandler::SetUPFNodeState(
   return;
 }
 
+/**
+ * Periodic messages about UPF session config
+ *
+ */
+void UpfMsgManageHandler::SetUPFSessionsConfig(
+    ServerContext* context, const UPFSessionConfigState* sess_config,
+    std::function<void(Status, SmContextVoid)> response_callback) {
+  auto& ses_config = *sess_config;
+  int32_t count    = 0;
+  conv_enforcer_->get_event_base().runInEventBaseThread([this, &count,
+                                                         ses_config]() {
+    for (int i = 0; i < ses_config.upf_session_state_size(); i++) {
```

Comment:
Might be nicer to iterate like
`for (auto upf_session : ses_config.upf_session_state)` unless you want the index.

Reply:
removed the index from for loop, and updated the for loop as you suggested.